### ISSUE TITLE
× binds stronger than →

### ISSUE THREAD

**Issue by AxelBoldt** [HoTT/book#949]
Section 1.5 should mention that × binds stronger than →, since numerous expressions in that section are ambiguous otherwise.

**Comment by mikeshulman** [comment]
Is there any context in which the opposite is true?  I would have assumed that mathematicians and computer scientists are all familiar with the fact that × binds stronger than →, and we don't want to sound too pedantically syntactic by mentioning binding precedence everywhere.

**Comment by AxelBoldt** [comment]
You are right, the binding of × is completely standard. Nevertheless I was surprised by it, since the binding of  \prod explained in 1.4 is utterly nonstandard.

**Comment by mikeshulman** [comment]
It's actually pretty common in type theory, but you're right that it's not the norm in the rest of mathematics.  That's why we explained it.
